src/test/serde_helpers.rs

```diff
@@ -0,0 +1,18 @@
+use serde::de::{Deserialize, DeserializeOwned, Deserializer};
```

Comment:
I didn't port all of our helpers over here but I thought it would be nice to have a central place to put methods like this to minimize clutter in test files.